### ISSUE TITLE
Model::findMultipleByIds() always returns a collection

### ISSUE THREAD

**Issue by leofeyer** [contao/contao#266]
**Affected version(s)**

Contao 4.4+

**Description**  

As noticed by @aschempp, the `Model::findMultipleByIds()` method always returns a collection, even if there are no models and the method should return `null`. All derived methods such as `FilesModel::findMultipleByIds()` behave correctly and return `null` if there are no models.

@contao/developers Any objections against fixing this in Contao 4.4?

**Comment by Toflar** [comment]
> @contao/developers Any objections against fixing this in Contao 4.4?

Yes, it's a BC break.

**Comment by leofeyer** [comment]
But it is also a bug.

**Comment by Toflar** [comment]
No it's not a bug. It's just not consistent but we're not going to break BC here just for the sake of consistency.

**Comment by fritzmg** [comment]
@Toflar the doc says
```
@return Model\Collection|null The model collection or null if there are no records
```
If it doesn't do what the doc always said, it's a bug, isn't it?

**Comment by aschempp** [comment]
It is also very unlikely that someone queries for multiple IDs and none of them are found in the DB, so I'm all for fixing this as well.

**Comment by Toflar** [comment]
Then the docs are wrong and should be fixed. What the method does is more important than the docs.
So I'm against it.

**Comment by leofeyer** [comment]
Fixed in fcd21f8003022c899d9ff01d8b0c32825df18672.
